pull_request_template.md

```diff
@@ -8,13 +8,19 @@
 [Discuss the impact of your changes on the project. This might include effects on performance, new dependencies, or changes in behaviour.]
 
 ## Checklist
-- [ ] My code adheres to the coding and style guidelines of the project.
-- [ ] I have performed a self-review of my own code.
-- [ ] I have extended current unit tests suite.
-- [ ] I have made corresponding changes to the documentation.
-- [ ] My changes generate no new warnings or new Sonar issues
-- [ ] I have passed the PR linting stage
-- [ ] I contacted my SRE colleague to update the configuration (if any changes)
+- [ ] **[JIRA]** I have linked the JIRA ticket to the PR.
+- [ ] **[CODE]** My code adheres to the coding and style guidelines of the project.
+- [ ] **[REVIEW]** I have performed a self-review of my own code.
+- [ ] **[DOCS]** I have made corresponding changes to the documentation.
+- [ ] **[SONAR]** My changes generate no new warnings or new Sonar issues.
+- [ ] **[LINT]** I have passed the PR linting stage.
+- [ ] **[SRE]** I contacted my SRE colleague to update the configuration (if any changes)
+- [ ] **[LABELS]** I added the relevant labels to the PR.
+- [ ] **[DEPS]** I have added/updated external dependencies/packages.
+- [ ] **[DEPS]** I have updated runtime version (i.e.: from node19 to node20).
+- [ ] **[TESTS]** I have extended current unit tests suite.
+- [ ] **[TESTS]** I have manually tested the changes.
+- [ ] **[TESTS]** I added e2e tests.

```

Comment:
I believe this only applies for a small group of repositories, can't we merge this and ut's into something like:

- [ ] I have extendedn the current test suite with:
	- [ ] unit tests
	- [ ] e2e tests
	- [ ] integration tests

Reply:
I agree somehow with @catalin-mihai-p5.
I think we can have one generic [DEPS] and one generic [TEST]

[DEPS] I have updated dependencies (runtime/packages...)
[TESTS] I have extended the tests suite (units, E2E, integrations)

The granular details are not applicable for all repositories.